raiden-dapp/tests/unit/app.spec.ts

```diff
@@ -6,29 +6,30 @@ import Vue from 'vue';
 import VueRouter from 'vue-router';
 import Vuex from 'vuex';
 import Vuetify from 'vuetify';
-import Mocked = jest.Mocked;
 import store from '@/store/index';
 import RaidenService from '@/services/raiden-service';
 import App from '@/App.vue';
+import Mocked = jest.Mocked;
 
 Vue.use(VueRouter);
 Vue.use(Vuex);
 Vue.use(Vuetify);
 
 describe('App.vue', () => {
-  let wrapper: Wrapper<App>;
   let vuetify: Vuetify;
   let router: Mocked<VueRouter>;
   let $raiden: RaidenService;
 
-  beforeEach(() => {
+  const createWrapper = (enableImprint = false): Wrapper<App> => {
+    if (enableImprint) process.env.VUE_APP_IMPRINT = 'https://custom-imprint.test';
```

Comment:
Nitpick: why not make the imprint just a string. This would also allow to have  a direct relation in the test case of the string you put in here as parameter and the one you expect to be set as `href` attribute.

Reply:
Done!